activities/tool-management/odoo.md

```diff
@@ -20,6 +20,10 @@ Our service provider is OpenWorx.
 
 The Operations coordinator takes care of user administration, reports bugs and is the contact person for the service provider.
 
+### Odoo aliases
+
+The aliases are necessary to send external emails directly to the right pipeline. The aliases are managed via the Google Workspace group admin panel and the backend of Odoo in developer mode with assets. The Operations Coordinator [has documentated in detail](https://docs.google.com/document/d/1KYmWUHCS-bA5Bqi2wQzl11D1QIxbtwr-Qtl2PEwKDw8/edit) how this works and is set up. There is also [an overview of aliases currently in use.](https://docs.google.com/spreadsheets/d/1jkyAFQuwspuLyJNc0zi_9Lw_xbHS4qcIavfAWMWTSIE/edit#gid=0)
+
 ## Privacy and GDPR
 
 See Openworx's:
```

Comment:
@demkodo Hey, this section doesn't explain anything about which of our sensitive data OpenWorkx has access to, or how or where it's processed (the stuff the Foundation's liable for under GDPR).

Reply:
I added more sections of their policy in Dutch, since the policy is only available in Dutch so translating it seems weird to do if the rest is in Dutch